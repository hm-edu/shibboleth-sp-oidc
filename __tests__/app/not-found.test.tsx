import { afterEach, beforeEach, describe, expect, it } from 'vitest';
import NotFound from '@/app/not-found';
import { cleanup, render, screen } from '@testing-library/react';

describe('NotFound', () => {
  beforeEach(() => {
    render(<NotFound />);
  });

  it('should exists', () => {
    expect(screen.findByTestId('not-found-illustration')).toBeDefined();
  });

  it('should match snapshot', () => {
    expect(screen.findByTestId('not-found-illustration')).toMatchSnapshot();
  });

  afterEach(() => {
    cleanup();
  });
});
