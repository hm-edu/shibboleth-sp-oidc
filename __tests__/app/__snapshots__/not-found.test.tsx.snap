// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NotFound > should match snapshot 1`] = `
<h2
  class="MuiTypography-root MuiTypography-h2 MuiTypography-alignCenter css-icizyq-MuiTypography-root"
>
  404 | Seite wurde nicht gefunden
</h2>
`;
